documentation-site/pages/blog/style-updates/index.mdx

```diff
@@ -0,0 +1,495 @@
+import {withStyle} from 'baseui';
+import {Block} from 'baseui/block';
+import {
+  Table,
+  StyledTable,
+  StyledHead,
+  StyledHeadCell,
+  StyledBody,
+  StyledRow,
+  StyledCell,
+} from 'baseui/table';
+
+import metadata from './metadata.json';
+import Layout from '../../../components/layout';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+<BlogImage
+  src="/static/images/blog/style-updates/baseblack.png"
+  alt="updated base web components with black highlights"
+  caption="Base Web has an updated look"
+/>
+
+## Migration Guide
```

Comment:
should we call this upgrade guide? ideally, this change shouldn't require code changes, right?